src/lib.rs

```diff
@@ -78,13 +78,16 @@
 //! ```rust,ignore
 //! #[macro_use]
 //! extern crate log;
-//! extern crate my_logger;
+//! extern crate env_logger;
 //!
 //! fn main() {
-//!     my_logger::init();
-//!
+//!     env_logger::init().unwrap();
+//!     
+//!     // Set the RUST_LOG level prior to running the binary
```

Comment:
This sounds like it says the next line `info!("starting up")` sets the RUST_LOG level. Also RUST_LOG contains more than just the level.

I would prefer to put this above the previous line and say something like:

```rust
// Select env_logger, one possible logger implementation
// (see https://doc.rust-lang.org/log/env_logger/index.html)
env_logger::init().unwrap();
```